src/stackdriver-nozzle/main.go

```diff
@@ -45,15 +48,24 @@ var (
 			Default(stackdriver.DefaultBatchDuration).
 			OverrideDefaultFromEnvar("BATCH_DURATION").
 			Duration()
+	boltDatabasePath = kingpin.Flag("boltdb-path", "Bolt Database path").
+				Default("cached-app-metadata.db").
+				OverrideDefaultFromEnvar("BOLTDB_PATH").
+				String()
 )
 
 func main() {
 	kingpin.Parse()
 
 	input := firehose.NewClient(*apiEndpoint, *username, *password, *skipSSLValidation)
-
 	sdClient := stackdriver.NewClient(*projectID, *batchCount, *batchDuration)
-	output := nozzle.Nozzle{StackdriverClient: sdClient}
+	cachingClient := caching.NewCachingBolt(input.EnsureCfClient(), *boltDatabasePath)
```

Comment:
Does the config story include the option to disable app lookup? We definitely want to do that for mvp, if not in a different story, here. It can really, really tax the API server. In situations where you're running lots of Apps, you would want to understand and give the operator the option to not do this.